lib/rucio/core/transfer.py

```diff
@@ -1353,7 +1353,7 @@ def update_transfer_priority(transfers_to_update, logger=logging.log):
             logger(logging.DEBUG, "Updated transfer %s priority in transfertool to %s: %s" % (transfer_id, priority, res['http_message']))
 
 
-def cancel_transfers(transfers_to_cancel, logger=logging.log):
+def cancel_transfers(transfers_to_cancel: dict[str, dict[str, int]], logger: LoggerFunction = logging.log) -> None:
```

Comment:
```suggestion
def cancel_transfers(transfers_to_cancel: dict[str,  Iterable[str]], logger: LoggerFunction = logging.log) -> None:
```

`FTS3Transfertool.cancel` expects a `Sequence[str]` as its input, and `transfer_ids` is just iterated over, so an `Iterable[str]` works. I think the docstring suggests that `transfer_ids` is a `set`, but I'm not sure. 

(If you want to annotate it as a `set`, please use `collections.abc.Set`): 

> Note that to annotate arguments, it is preferred to use an abstract collection type such as [collections.abc.Set](https://docs.python.org/3/library/collections.abc.html#collections.abc.Set) rather than to use [set](https://docs.python.org/3/library/stdtypes.html#set) or [typing.Set](https://docs.python.org/3/library/typing.html#typing.Set).

https://docs.python.org/3/library/typing.html#typing.Set